worlds/shivers/__init__.py

```diff
@@ -168,6 +215,7 @@ def fill_slot_data(self) -> dict:
         return {
             "storageplacements": self.storage_placements,
             "excludedlocations": {str(excluded_location).replace('ExcludeLocations(', '').replace(')', '') for excluded_location in self.multiworld.exclude_locations.values()},
+            "ixupicapturesneeded": {self.options.ixupi_captures_needed.value},
```

Comment:
Can we add at least `FrontDoorUsable` if not all other options we do not have here. Would rather make sure we have the options available even if we do nothing with them (could be useful for the poptracker as well).

Also renaming these to be camel case. I know that means an update on the client, but it's minimal.


Also pretty certain the {} aren't needed
```suggestion
            "IxupiCapturesNeeded": self.options.ixupi_captures_needed.value,
```

Reply:
Front door added. 

As far as all the options I was told at an earlier point in time not to add to slot data if its not going to be used.

Excluded locations is on the to do list. It is for when a location is excluded that is locking a pot. For example: if the player excludes Chinese Solitaire they shouldn't have to do that puzzle to obtain the pot piece.